core/nwb.file.yaml

```diff
@@ -450,6 +450,12 @@ groups:
     dtype: text
     doc: Age of subject. Can be supplied instead of 'date_of_birth'.
     quantity: '?'
+    attributes:
+      - name: reference
+        doc: "Age is with reference to this event. Can be 'birth' or 
+          'gestational'. If reference is omitted, 'birth' is implied."
```

Comment:
Is it possible to set default values for this at the schema level? I.e., `default: 'birth'` since `required: false`?

Reply:
I was thinking about this. I suppose it is possible. I wanted to make required: false so that this would be backwards compatible. But if you are writing new data perhaps it would be better to make it explicit by having the default behavior be to write "birth" instead of omitting the arg. I'd be fine either way. Thoughts from the team? Should we give this a default value of "birth"?

Reply:
Defining a default value of `birth` makes sense to me

Reply:
Defining a default value of `birth` makes sense to me too